tensorflow_datasets/structured/bbbp_test.py

```diff
@@ -0,0 +1,23 @@
+"""
+Test for BBBP Dataset.
+"""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+from tensorflow_datasets import testing
+from tensorflow_datasets.structured import bbbp
+
+
+class BbbpTest(testing.DatasetBuilderTestCase):
+  DATASET_CLASS = bbbp.Bbbp
+  SPLITS = {
+      "train": 1
+  }
+
+  DL_EXTRACT_RESULT = "BBBP.csv"
```

Comment:
I don't think this line is needed if there is only one extract path.

Reply:
I ran the test without it, the test fails saying the fake data file couldn't be found.